eng/tools/publish-to-npm.ps1

```diff
@@ -12,7 +12,7 @@ param (
 )
 
 function replaceText($oldText,$newText,$filePath){
-    $content = Get-Content -Path $filePath
+    $content = Get-Content -Path $filePath -Raw
     $newContent = $content -replace $oldText,$newText
     if((-join $newContent) -ne (-join $content)){
         Set-Content -Path $filePath -Value $newContent
```

Comment:
Generally you also need to pair the write up with the `-NoNewLine` option to avoid changing the content.